pkg/sql/set_cluster_setting.go

```diff
@@ -193,6 +195,18 @@ func (p *planner) getAndValidateTypedClusterSetting(
 }
 
 func (n *setClusterSettingNode) startExec(params runParams) error {
+
+	if strings.Contains(n.name, "sql.defaults") {
+		params.p.BufferClientNotice(
+			params.ctx,
+			errors.WithHintf(
+				pgnotice.Newf("Setting global default %s`;", n.name),
```

Comment:
nit: errors start with lowercase, so "setting global default". remove the semicolon `;` as well